packages/components/pie-divider/src/divider.scss

```diff
@@ -1,22 +1,52 @@
 @use '@justeattakeaway/pie-css/scss' as p;
 
 .c-divider {
-    --divider-bg-color: var(--dt-color-divider-default);
-    --divider-width: 100%;
-    --divider-height: 1px;
+  --divider-bg-color: var(--dt-color-divider-default);
+  --divider-width: 100%;
+  --divider-height: 1px;
+  --divider-label-max-width: 90%;
 
+  width: var(--divider-width);
+
+  &,
+  hr {
     margin: 0;
     border: 0;
-    width: var(--divider-width);
-    height: var(--divider-height);
     background-color: var(--divider-bg-color);
+  }
 
-    &.c-divider--inverse {
-        --divider-bg-color: var(--dt-color-divider-inverse);
+  &:not(.c-divider--labelled) {
+    height: var(--divider-height);
+  }
+
+  &.c-divider--labelled {
```

Comment:
One thing I just noticed testing against storybook against the long text variant – in Figma it says the dividers should have a minimum width on each side of 16px, but atm, it looks like the `hr`'s on each side are 12px width?